server/src/pogo-accounts.routes.ts

```
@@ -110,7 +110,7 @@ pogoAccountsRouter.delete("/:id", async (req, res) => {
         const result = await collections.pogoAccounts.findOneAndDelete(query)
 
         if (typeof result !== null) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)